polynote-kernel/src/main/scala/polynote/kernel/logging/Logging.scala

```diff
@@ -53,7 +59,11 @@ object Logging {
         out.synchronized {
           out.print(Red)
           out.print(errorPrefix)
-          out.println(msg)
+          out.print(msg)
+          if (location.file != "")
```

Comment:
Now printing out locations of where the log is coming from, when there's an error logged. It was getting confusing to find the place where the error was logged.

Reply:
I like this logging framework we've slowly built up... 😜

Reply:
I mean, I wouldn't be against using slf4j (via `zio-logging`) instead, except for the fact that Spark breaks it horribly. I don't hate logging per se, I just hate wrestling with it and I know it will be a wrestle if we try to use some existing logger. Spark will seek it out and destroy it.